### DISCUSSION TITLE
Use target _blank by default

### DISCUSSION THREAD

**Discussion by kamieniarz** [squidfunk/mkdocs-material#5534] [Q&A] [answered]
I'd like to use target _blank in all links in text. Generally looks like I'd have to do sth like this in all my *.md files: `https://google.com` -> `<a href="https://google.com" target="_blank">https://google.com</a>` which is kinda tricky and would take a lot of time.
I tried to apply custom script (the one below) but it's too general - it opens nav links and enything possible in new tab
```
window.addEventListener('DOMContentLoaded', (event) => {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    links[i].setAttribute('target', '_blank');
  }
});
```

Is there any way I could set all links from *.md files to open in new tab? Just these likns without the need to edit them like in my first example

**Comment by squidfunk** [comment, marked as answer]
Duplicate:

- https://github.com/squidfunk/mkdocs-material/discussions/3141
